src/Components/Aspire.Microsoft.Azure.StackExchangeRedis/AzureStackExchangeRedisSettings.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Azure.Core;
+
+namespace Aspire.Microsoft.Azure.StackExchangeRedis;
+
+/// <summary>
+/// Provides the client configuration settings for connecting to an Azure Cache for Redis using StackExchange.Redis.
+/// </summary>
+public sealed class AzureStackExchangeRedisSettings
```

Comment:
This should inherit from `StackExchangeRedisSettings` instead. And only add the `Credential` property.